### ISSUE TITLE
Check:imports command wrong reports when use question mark in doc blocks

### ISSUE THREAD

**Issue by alikhosravidev** [imanghafoori1/php_token_analyzer#21]
PHP: v8.1
Laravel: v9
microscope: v1.0.318
Os: Pop Os

Autoload psr4:
"autoload": { "files": [ "app/helpers.php" ], "psr-4": { "App\\": "app/", "Database\\Factories\\": "database/factories/", "Database\\Seeders\\": "database/seeders/" } },

```php
<?php

namespace App\Http\Controllers;

use App\Models\User;
use Carbon\Carbon;
use Illuminate\Routing\Controller as BaseController;

/**
 * @property ?Carbon $created_at
 */
class Controller extends BaseController
{
    /**
     * @var ?User $user
     */
    public function index($user)
    {
        return $user;
    }
}

```

Reported message:
![image](https://github.com/imanghafoori1/php_token_analyzer/assets/49164621/6e917bb4-317a-43ad-b5ba-78344ef16f87)

**Comment by imanghafoori1** [comment]
fixed
